server/app/services/databaseConnection/DBConnection.java

```diff
@@ -0,0 +1,109 @@
+package services.databaseConnection;
+
+import play.Configuration;
+import play.api.Play;
+
+import java.sql.*;
+import java.util.ArrayList;
+
+/**
+ * Created by sabinapokhrel on 11/7/16.
+ */
+
+
+public class DBConnection {
+  private Configuration configuration = Play.current().injector().instanceOf(Configuration.class);
+  /**
+   * Method to connect to database
+   * @return
+   */
+  Connection connectDB() {
+
+    /* These variables must be made configurable */
+    String host = configuration.getString("db-conf.host");
+    String dbName = configuration.getString("db-conf.dbName");
+    String username = configuration.getString("db-conf.username");
+    String password = configuration.getString("db-conf.password");
+
+    try {
+      /* Dynamically load the JDBC driver */
+      Class.forName("org.postgresql.Driver"); //load the driver
+
+      /* Connect to the Postgres database */
+      Connection db = DriverManager.getConnection(
+        "jdbc:postgresql://" + host + "/" + dbName,
+        username,
+        password); //connect to the db
+      return db;
+    } catch (Exception e) {
+      e.printStackTrace();
+    }
+    return null;
+  }
+
+  /**
+   * SQL statement to add the question to database
+   * @param question
+   */
+  public void addQuestionToDB(String question) {
+    DBConnection DBConnection = new DBConnection();
+    Connection connection = DBConnection.connectDB();
+
+    DBConnection.createDatabase(connection);
+
+    try{ try {
+      PreparedStatement stmt = connection.prepareStatement("insert into questions (question) values (?)");
+      //stmt.setInt(1, 1);
+      stmt.setString(1, question);
+      stmt.executeUpdate();
+    } finally {
+      connection.close();
+    }
+    }catch (Exception e) {
+      e.printStackTrace();
+    }
+  }
+
+  /**
+   * Creates a table questions if it does not already exists
+   * @param connection
+   */
+  public void createDatabase(Connection connection){
+    try{
+      PreparedStatement preparedStatement = connection.prepareStatement(
+        "CREATE TABLE questions(qno SERIAL PRIMARY KEY, question TEXT NOT NULL);");
+      preparedStatement.executeUpdate();
+    }catch (Exception e) {
+      //e.printStackTrace();
```

Comment:
Please `throw` an exception here, do not hide errors.

Use other method to check if the `questions` table is already existed.